components/admin_console/admin_user_card/admin_user_card.tsx

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {Client4} from 'mattermost-redux/client';
+
+import ProfilePicture from 'components/profile_picture';
+import * as Utils from 'utils/utils.jsx';
+
+import './admin_user_card.scss';
+
+type BulletProps = {
+    user: {
```

Comment:
Can these user props use the `UserProfile` type from `mattermost-redux/types/users`? If you get complains from the unit tests not passing in properly typed objects, you can use the `TestHelper.getUserMock` method to construct proper user objects.

Reply:
Hi @hmhealey  I already change it to use type UserProfile.